ci/steps.yml

```diff
@@ -108,12 +108,28 @@ steps:
         fi
         aqt list-qt $(HOST) $(TARGET) $ext --arch $(QT_VERSION)                 # print architectures for version/host/target
         aqt list-qt $(HOST) $(TARGET) $ext --spec "$(SPEC)" --arch latest
+
+        # list-src/doc/example
+        aqt list-src $(HOST) $(QT_VERSION)                                      # print source archives available for host/desktop/version
+        aqt list-doc $(HOST) $(QT_VERSION)                                      # print docs archives available for host/desktop/version
+        aqt list-doc $(HOST) $(QT_VERSION) --modules                            # print docs modules available for host/desktop/version
+        aqt list-example $(HOST) $(QT_VERSION)                                  # print example archives available for host/desktop/version
+        aqt list-example $(HOST) $(QT_VERSION) --modules                        # print example modules available for host/desktop/version
```

Comment:
This code demonstrates how the new `list-src`, `list-doc`, and `list-example` commands work. I am open to suggestions on how they should be designed; I am not certain that this was the right choice.

There are two obvious options here:
1. list modules by default, and use `--archives` to print archives
2. list archives by default, and use `--modules` to print modules

I used option 2 because I wanted the interface to be consistent across `list-src`, `list-doc`, and `list-example`. The `list-src` command cannot print modules at all, so if I had used option 1, `list-src mac 6` would print an error message, and `list-src mac 6 --archives` would print archives; I thought this would be surprising.